test/PartTest.php

```diff
@@ -205,4 +205,20 @@ public function testSetContentRaisesInvalidArgumentExceptionForInvalidContentTyp
         $this->expectException(Mime\Exception\InvalidArgumentException::class);
         $part->setContent($content);
     }
+
+    public function testBinaryPart()
```

Comment:
Give this a better name: `testBinaryPartsAreCreatedAndParsedWithoutLoss()` or similar.